src/main/resources/liquibase/module/scripts/v-0.0.1/2023-08-14--00-00-remove-queue-item-size-column.xml

```diff
@@ -0,0 +1,8 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+  <changeSet id="2023-08-14--00-00-remove-queue-item-size-column" author="ncovercash">
+    <dropColumn tableName="dataImportQueueItems" columnName="size" />
```

Comment:
would like to propose deleting this column in the initial script where the table is created instead of adding an extra script since this table has not been included in any module release yet and therefore this table is still under development

Reply:
I'd prefer to keep it together, at least until a later point in DITF's work; otherwise, we'd have a lot of issues with all our local dev environments + Rancher